Tests/UnitTests/Sources/GREYAnalyticsTest.m

```diff
@@ -15,13 +15,12 @@
 //
 
 #import <EarlGrey/GREYAnalytics.h>
-#import <EarlGrey/GREYAnalyticsDelegate.h>
 #import <EarlGrey/NSString+GREYAdditions.h>
 
 #import "GREYBaseTest.h"
 #import "GREYExposedForTesting.h"
 
-NSString *gTestDelegateBundleId;
+NSString *gTestDelegateBundleID;
```

Comment:
Please remove this global and add it as a property in GREYAnalyticsTestDelegate:
@property(nonatomic, strong) NSString *bundleID;